### DISCUSSION TITLE
How can I get the ASIN for a deleted SKU?

### DISCUSSION THREAD

**Discussion by ProNotion** [abuzuhri/Amazon-SP-API-CSharp#379] [Q&A]
What are the options for me to retrieve the ASIN associated with a deleted SKU if I have the original SKU code and the FNSKU?

Thanks in advance.

**Comment by J-W-Chan** [comment]
> What are the options for me to retrieve the ASIN associated with a deleted SKU if I have the original SKU code and the FNSKU?
> 
> Thanks in advance.

you should create a issue to [sp-api official GitHub](https://github.com/amzn/selling-partner-api-docs/issues)，or create a ticket from develop support(https://developer.amazonservices.com/support).

**Reply by ProNotion** [reply]
I did start a discussion on there but its been met with no response so thought I would ask here on the off chance that someone knew.  I didn't want to create an issue as such as it might not be an issue. It may just be that I haven't been able to find the solution but the option is there.

**Comment by abuzuhri** [comment]
depending on my experience there is method for this only when you get offerforASIN you will get message that there is no ASIN in this market place , but maybe other person has another solution

**Comment by ProNotion** [comment]
If there are any historical orders for the SKU you can at least retrieve the ASIN from the order item. Assuming the ASIN is still live then you can get additional information relating to the item from the listing.
